packages/rollup-contracts/test/rollup-list/L1ToL2TransactionQueue.spec.ts

```diff
@@ -0,0 +1,152 @@
+import '../setup'
+
+/* External Imports */
+import { getLogger } from '@eth-optimism/core-utils'
+import { createMockProvider, deployContract, getWallets } from 'ethereum-waffle'
+
+/* Internal Imports */
+import { DefaultRollupBlock } from './RLhelper'
+
+/* Logging */
+const log = getLogger('l1-to-l2-tx-queue')
```

Comment:
```suggestion
const log = getLogger('l1-to-l2-tx-queue', true)
```

Reply:
Oh yes good point, come to think of it I think there is an `assertRevertsAsync` which accepts an EVM revert message to check!